frontend/leads-list/table/lead-services-cell.component.tsx

```diff
@@ -0,0 +1,274 @@
+import React from "react";
+import { useCss } from "kremling";
+import preventiveHealthIcon from "../../../icons/148705-essential-collection/svg/003-caduceus.svg";
+import nutritionIcon from "../../../icons/148705-essential-collection/svg/009-fruit.svg";
+import immigrationIcon from "../../../icons/148705-essential-collection/svg/027-usa-today-2.svg";
+import financeIcon from "../../../icons/148705-essential-collection/svg/038-money-2.svg";
+import workerRightsIcon from "../../../icons/148705-essential-collection/svg/064-hand-holding-up-a-wrench.svg";
+import familySupportIcon from "../../../icons/148705-essential-collection/svg/061-family-house.svg";
+import communityEngagementIcon from "../../../icons/148705-essential-collection/svg/055-megaphone.svg";
+import focusGroupsIcon from "../../../icons/148705-essential-collection/svg/070-lightbulb-idea.svg";
+
+export default function LeadServicesCell(props: LeadServicesCellProps) {
```

Comment:
This component is more complicated than it needs to be. I'm going to simplify it.